Added README and LEIAME for PT-BR Version (#1)
code-club-brasil/bingo: README.md

# BinGo

Portuguese Version in [LEIAME.md](LEIAME.md)

## Objective

> This repo has the objective to create a Bingo game written in [GoLang](https://golang.org/), using peer to peer learning techniques to contribute and teach each of the willing members to lern more about the language and advance in one's studies about Software Developing and Go.

Reviewer: ```suggestion
This repo host a Bingo game is written in [GoLang](https://golang.org/), using peer to peer learning techniques to contribute and teach each of the willing members to learn more about the language and advance in one's studies about Software Developing.
```
